keras/utils/torch_utils_test.py

```diff
@@ -25,24 +34,31 @@ def __init__(self, *args, **kwargs):
         self.fc2 = layers.Dense(2)
 
     def call(self, x):
-        return self.fc(self.fc2(x))
+        return self.fc2(self.fc(x))
 
+@pytest.mark.skipif(
+    backend.backend() != "torch", reason="Requires torch backend"
+)
+class TorchUtilsTest(testing.TestCase, parameterized.TestCase):
 
-class TorchUtilsTest(testing.TestCase):
-    @pytest.mark.skipif(
-        backend.backend() != "torch", reason="Requires torch backend"
+    @parameterized.parameters(
+        {'use_batch_norm': False},
+        {'use_batch_norm': True},
     )
-    def test_basic_usage(self):
-        model = Classifier()
-        self.assertEqual(len(model.layers), 1)
-        self.assertEqual(len(model.trainable_weights), 2)
+    def test_basic_usage(self, use_batch_norm):
+        model = Classifier(use_batch_norm)
+        self.assertEqual(len(model.layers), 2)
+        # Linear - Weights, bias, BN - beta, gamma
+        fc_trainable_count = 2
+        if use_batch_norm:
```

Comment:
Probably need to override `load_own_weights` to support BatchNorm.

Reply:
I tried the `store.keys()` in `load_own_variables`, but it doesn't contain the `BatchNorm` Mean and Variance variables since they are not part of `model.parameters`.  So not sure what's the best way to capture or register it since `torch` keeps it separate in the `BatchNorm` layer.  Can you provide your thoughts on the comment here - here - https://github.com/keras-team/keras/issues/18403#issuecomment-1734733059